src/chord.py

```diff
@@ -0,0 +1,101 @@
+from os import O_TRUNC
+import parselmouth
+import math
+
+
+class Sound:
+    def __init__(self, note, timestamp=0, duration=0):
+        self.note = note
+        self.timestamp = timestamp
+        self.duration = duration
+
+    @property
+    def symbol(self):
+        if self.note == None:
+            return 'None'
+        #           0    1     2    3     4
+        symbols = ['C', 'C#', 'D', 'D#', 'E',
+                   # 5   6     7    8     9    10    11
+                   'F', 'F#', 'G', 'G#', 'A', 'A#', 'H']
+        return symbols[self.note]
+
+    @property
+    def note(self):
+        return self._note
+
+    @note.setter
+    def note(self, note):
+        self._note = note % 12 if note else note
+
+    def __str__(self):
+        return f"{self.timestamp}: {self.symbol}"
+
+    def __eq__(self, other):
+        return self.note == other.note
+
+    __repr__ = __str__
+
+
+def frequency_to_note(frequency):
+    '''
+    A4 440 Hz
+    '''
+    if frequency == 0:
+        return None
+    return round(12*math.log2(frequency/440.0)+45) % 12
+
+
+class Sounds:
+    def __init__(self):
+        self.sounds = []
+        self._get_sounds_from_pitch()
+
+    def __str__(self):
+        return "".join([str(sound) for sound in self.sounds])
+
+    def _get_sounds_from_pitch(self):
```

Comment:
This is a mockup and there's no way to tell that other than guessing. Rename the class to SoundsMockup or follow the new convention, remove the whole Sounds class and stay only with get_sounds_from_file_mockup(file). The usage should be the same as the original get_sounds_from_file(file) function, that's why file should be in the arguments.

Reply:
Ok, so as far as I understand this is bc you included my commits only accidentally. Still, mockup can still be useful, but it should be named mockup